backstage/values.yaml

```diff
@@ -152,8 +152,8 @@ appConfig:
   sentry:
     organization: spotify
   techdocs:
-    storageUrl: https://demo.example.com/api/techdocs/static/docs
-    requestUrl: https://demo.example.com/api/techdocs
+    generators:
```

Comment:
We won't need to specify credentials via app config to access s3 (AFAIK Jussi is configuring policies for the backend pod role?) but for other publishers secrets would need to be set in the `backstage-values` secret which I believe is manually created.

Reply:
No need for creds, they'll be pulled from the instance

Reply:
Publishing would be handled outside of backstage at least for contentful (and ideally for everyone else as well)

Reply:
It's a little confusing semantically but the publisher config is also used in the read-only path (builder: 'external'). So if we wanted to say read from GCS we'd need to configure the secret in the publisher config. _I think_. But this is just an aside that I think we could support that if / when we came to it.

Reply:
Yup, it uses the same config but all AWS, GCP and Azure should be able to be set up with env variables so we don't need to add anything on this side I think